READMEs/publish-flow-credentials.md

```diff
@@ -0,0 +1,52 @@
+<!--
```

Comment:
Is a readme truly necessary? If so, can you please make sure it is tested?

Reply:
Also, could you please wait for https://github.com/oceanprotocol/ocean.py/pull/1393 to be merged first and adapt your code to use kwargs? I've added more flexibility to adding parameters to all these functions and you can use kwargs.get("credentials") when needed instead of adding the kwarg manually

Reply:
yes, it is :)  people are asking me how to use credentials (that's why I started this PR).
We already have tests for credentials, so we should be fine